src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -110,10 +116,14 @@ public override bool Execute()
                                                                 fileSystem,
                                                                 ServiceProvider.GetService<IConsole>(),
                                                                 result);
-                        writer.WriteReport();
+                        var path = writer.WriteReport();
+                        var metadata = new Dictionary<string, string> { ["Format"] = format };
+                        coverageReportPaths.Add(new TaskItem(path, metadata));
```

Comment:
Please update msbuild documentation with the format of the output and an example of how it can be used

Reply:
I already added a new paragraph in MSBuildIntegration.md and provided an example of how it can be used with ReportGenerator. Is there something missing or that I should clarify?

Reply:
@tonerdo you mean this https://github.com/coverlet-coverage/coverlet/pull/932/files#diff-318e71585840883ebf9a0b9a112c5ea3?

Reply:
My bad, missed that bit